hackathon_site/dashboard/frontend/src/components/inventory/ProductOverview/ProductOverview.test.js

```diff
@@ -65,15 +69,37 @@ describe("<EnhancedAddToCartForm />", () => {
 
     test("requestFailure is passed in and rendered properly", () => {
         const requestFailure = { message: "Failed" };
-        const availableQuantity = "4";
+        const quantityAvailable = 4;
 
         const { getByText } = render(
             <EnhancedAddToCartForm
                 requestFailure={requestFailure}
-                availableQuantity={availableQuantity}
+                quantityAvailable={quantityAvailable}
             />
         );
 
         expect(getByText(requestFailure.message)).toBeInTheDocument();
     });
+
+    test("button and select are disabled if quantityAvailable is 0", () => {
+        const { getByText, getByTestId } = render(
+            <EnhancedAddToCartForm quantityAvailable={0} />
+        );
+
+        const button = getByText("Add to cart");
+        const select = getByTestId("qtySelect");
```

Comment:
Because the test id is on the wrong element, this is actually giving you the button.

Once you add a label to the select and input label, you can use `getByLabelText("Qty")`. However since this isn't an actual html input (under the hood, it isn't actually a `<select>`), you won't be able to use `.toBeDisabled()`. You could check for the `aria-disabled="true"` tag, or the `.Mui-disabled` class.